src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx

```diff
@@ -86,43 +100,49 @@ function TransactionSubmittedContent({
 
   return (
     <Wrapper>
-      <Section>
-        <RowBetween>
-          <div />
-          <CloseIcon onClick={onDismiss} />
-        </RowBetween>
-        <ConfirmedIcon>
-          <ArrowUpCircle strokeWidth={0.5} size={90} color={theme.primary1} />
+      <Section inline={inline}>
+        {!inline && (
+          <RowBetween>
+            <div />
+            <CloseIcon onClick={onDismiss} />
+          </RowBetween>
+        )}
+        <ConfirmedIcon inline={inline}>
+          <ArrowUpCircle strokeWidth={0.5} size={inline ? '40px' : '90px'} color={theme.primary1} />
         </ConfirmedIcon>
         <AutoColumn gap="12px" justify={'center'}>
-          <Text fontWeight={500} fontSize={20}>
-            Transaction Submitted
+          <Text fontWeight={500} fontSize={20} textAlign="center">
+            <Trans>Transaction Submitted</Trans>
           </Text>
           {chainId && hash && (
+            // <ExternalLink href={getExplorerLink(chainId, hash, ExplorerDataType.TRANSACTION)}>
             <ExternalLink href={getEtherscanLink(chainId, hash, 'transaction')}>
               <Text fontWeight={500} fontSize={14} color={theme.primary1}>
                 {/* View on Etherscan */}
-                {getExplorerLabel(chainId, hash, 'transaction')}
+                {/* <Trans>View on Explorer</Trans> */}
+                <Trans>{getExplorerLabel(chainId, hash, 'transaction')}</Trans>
```

Comment:
I don't think it makes sense to translate this, as it won't be known at compile time.
Or will it?

Reply:
yes, true

Reply:
but we can add it later via id's in the language files, i'll keep it around

Reply:
What do you mean by via ids?

Reply:
inside `src/locales` each locale file has `id`s set for translating. so i meant we can just add these translations later
e.g

```
#: src/components/AccountDetails/index.tsx
msgid "(clear all)"
msgstr "(clear all)"
```